commit 60376cb2e64dc043dce660da823a737e5f44e577 (refs/changes/60/3460/3)
Author: Mike Gerdts <mike.gerdts@joyent.com>
Date:   2018-02-23T20:55:50+00:00 (1 year, 7 months ago)
    
    OS-6689 illumos-extra.manifest not regenerated when projects/illumos-extra/manifest changes
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/Makefile b/Makefile
index 8b79c2f5..00098ebb 100644
--- a/Makefile
+++ b/Makefile
@@ -206,7 +206,8 @@ $(MCPROTO)/illumos.mancheck.conf: projects/illumos/mancheck.conf | $(MCPROTO)
 $(BOOT_MPROTO)/illumos.manifest: projects/illumos/manifest | $(BOOT_MPROTO)
 	cp projects/illumos/boot.manifest $(BOOT_MPROTO)/illumos.manifest
 
-$(MPROTO)/illumos-extra.manifest: 1-extra-stamp | $(MPROTO)
+$(MPROTO)/illumos-extra.manifest: 1-extra-stamp \
+    projects/illumos-extra/manifest | $(MPROTO)
 	gmake DESTDIR=$(MPROTO) DESTNAME=illumos-extra.manifest \
 	    -C projects/illumos-extra manifest; \
 
